srcpkgs/cloud-utils/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'cloud-utils'
+pkgname=cloud-utils
+version=0.33
+revision=1
+archs="x86_64"
+#wrksrc=
+#create_wrksrc=yes
+build_style=gnu-makefile
+#configure_args=""
+#make_build_args=""
+#make_install_args=""
+#conf_files=""
+#make_dirs="/var/log/dir 0755 root root"
+hostmakedepends=""
+makedepends=""
+depends=""
+short_desc="This package provides a useful set of utilities for interacting with a cloud"
```

Comment:
`xlint` (from the package `xtools`) reports:
```
template:17: short_desc should be less than 72 chars
```

Reply:
resolved it.